primer-miso/Makefile.wasm32

```diff
@@ -0,0 +1,26 @@
+# NOTE:
+#
+# This Makefile assumes you're using the `nix develop .#wasm` shell.
+
+CABAL = wasm32-wasi-cabal
+
+build:	configure
```

Comment:
Ahh, I was wondering where that came from. That seems to be something new, because the existing Makefiles have been around for over a year and have worked fine until recently.

Reply:
I don't see anything here that should have changed the behaviour. I figured I hadn't personally hit this until now because I'd never even used the Wasm shell.

Reply:
Fixed.